documentation/sphinx/source/api-c.rst

```diff
@@ -295,6 +295,12 @@ See :ref:`developer-guide-programming-with-futures` for further (language-indepe
 
    |future-get-return1| |future-get-return2|.
 
+.. function:: fdb_error_t fdb_future_get_int64(FDBFuture* future, int64_t* out)
```

Comment:
The `fdb_future_get_version` function just above can be removed from the documentation, and references to it elsewhere in this file should be updated to refer to this function.